homeassistant/components/mvglive/sensor.py

```diff
@@ -68,29 +70,27 @@
 )
 
 
-def setup_platform(
+async def async_setup_platform(
     hass: HomeAssistant,
     config: ConfigType,
     add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the MVGLive sensor."""
-    add_entities(
-        (
-            MVGLiveSensor(
-                nextdeparture.get(CONF_STATION),
-                nextdeparture.get(CONF_DESTINATIONS),
-                nextdeparture.get(CONF_DIRECTIONS),
-                nextdeparture.get(CONF_LINES),
-                nextdeparture.get(CONF_PRODUCTS),
-                nextdeparture.get(CONF_TIMEOFFSET),
-                nextdeparture.get(CONF_NUMBER),
-                nextdeparture.get(CONF_NAME),
-            )
-            for nextdeparture in config[CONF_NEXT_DEPARTURE]
-        ),
-        True,
-    )
+    sensors = [
+        MVGLiveSensor(
+            hass,
+            nextdeparture.get(CONF_STATION),
+            nextdeparture.get(CONF_DESTINATIONS),
+            nextdeparture.get(CONF_LINES),
+            nextdeparture.get(CONF_PRODUCTS),
+            nextdeparture.get(CONF_TIMEOFFSET),
+            nextdeparture.get(CONF_NUMBER),
+            nextdeparture.get(CONF_NAME),
+        )
```

Comment:
We don't pass `CONF_DIRECTIONS` to the new sensor, is it no longer supported?
If so, we should mark the `CONF_DIRECTIONS` as deprecated and remove it from the documentation.